harfbuzz-sys/Cargo.toml

```diff
@@ -37,12 +37,13 @@ optional = true
 features = ["dwrite"]
 
 [dependencies.freetype-sys]
-version = "0.20.1"
+version = "0.21.0"
 optional = true
 
 [features]
 default = ["coretext", "directwrite", "freetype"]
 bundled = []
+freetype-bundled = ["freetype-sys/bundled"]
```

Comment:
I don't thin you need another feature for this. Features are additive, so one should just need to make sure that freetype-sys is has the bundled feature when compiling. This is something that someone using rust-harfbuzz can do without having to expose a new feature here.

Reply:
Should the `freetype-bundled` feature be exposed in the `harfbuzz` wrapper only then, since features of subdependencies can't be specified?

Reply:
I don't believe it's possible in CLI to enable features of sub dependencies ([it is possible in `Cargo.toml`, though!](https://ntietz.com/blog/til-enabling-features-on-transitive-dependencies-rust/))